### PR TITLE
router statistics - sort tunnels by traffic

### PR DESCRIPTION
**Pull request by RetroPooh**

### CODE REVIEW

File: retroshare-gui/src/gui/statistics/TurtleRouterDialog.cpp [hunk 1]
```diff
@@ -170,8 +173,20 @@ void TurtleRouterDialog::updateTunnelRequests(	const std::vector<std::vector<std
...
-
-		QString str = tr("Tunnel id") + ": " + QString::fromUtf8(tunnels_info[i][0].c_str()) + "\t" + tr("Speed") + ":  " + QString::fromStdString(tmp) + "\t " + tr("last transfer") + ": " + QString::fromStdString(tunnels_info[i][4])+ "\t" + QString::fromUtf8(tunnels_info[i][2].c_str()) + " -> " + QString::fromUtf8(tunnels_info[i][1].c_str());
+		
+		float num2 = strtoull(tunnels_info[i][6].c_str(), NULL, 0);
+		char tmp2[100] ;
+		std::string units2[4] = { "B","KB","MB","GB" } ;
+		int k2=0 ;
+		while(num2 >= 800.0f && k2<4)
+			num2 /= 1024.0f,++k2;
+		sprintf(tmp2,"%3.2f %s",num2,units2[k2].c_str()) ;
```
Comment (PhenomRetroShare) [inline retroshare-gui/src/gui/statistics/TurtleRouterDialog.cpp:183]: Maybe you can use:
/retroshare-gui/src/util/misc.cpp:36
QString misc::friendlyUnit(float val)

**Comment by defnax** [comment]
any news for merge @csoler @PhenomRetroShare @RetroPooh ?

**Comment by defnax** [comment]
he has nice ideas

**Comment by RetroPooh** [comment]
Fixed here: https://github.com/RetroShare/RetroShare/pull/2751 + https://github.com/RetroShare/libretroshare/pull/108

**Pull request closed without merging**